tests/test_identity_map_client_unit_tests.py

```diff
@@ -30,3 +33,25 @@ def test_get_datetime_utc_iso_format_timestamp(self):
             iso_format_timestamp = get_datetime_utc_iso_format(timestamp)
             self.assertEqual(expected_timestamp, iso_format_timestamp)
 
+    @patch('uid2_client.identity_map_client.make_v2_request')
+    @patch('uid2_client.identity_map_client.post')
+    @patch('uid2_client.identity_map_client.parse_v2_response')
+    def test_identity_buckets_request(self, mock_parse_v2_response, mock_post, mock_make_v2_request):
+        expected_timestamp = "2024-07-02T14:30:15.123456+00:00"
+        expected_req = json.dumps({"since_timestamp": get_datetime_utc_iso_format(dt.datetime.fromisoformat(expected_timestamp))}).encode()
```

Comment:
extra safety nit: could you add a second/separate expectation to compare with the hardcoded iso string output (instead of relating on the computed result from get_datetime_utc_iso_format) to ensure it is what we expect?